util/manual_utils.ts

```diff
@@ -26,3 +27,7 @@ export async function getTableOfContents(
 export function getFileURL(version: string, path: string) {
   return `${basepath}${version}/docs${path}.md`;
 }
+
+export function getDocURL(version: string, path: string) {
+  return `${docpath}${version}/docs${path}.md#readme`;
```

Comment:
remove `#readme`

Reply:
I don't see any harm here but as you wish 🙂